src/EventListener/ControllerListener.php

```diff
@@ -60,6 +60,20 @@ public function onKernelController(KernelEvent $event)
         $classConfigurations = $this->getConfigurations($this->reader->getClassAnnotations($object));
         $methodConfigurations = $this->getConfigurations($this->reader->getMethodAnnotations($method));
 
+        if (method_exists($method, 'getAttributes')) {
```

Comment:
```suggestion
        if (80000 >= \PHP_VERSION_ID) {
```

OPcache can optimize PHP version conditions and we can find this piece easily if we build a release without PHP 7 support one day.